sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserFacadeEjb.java

```diff
@@ -245,25 +247,31 @@ public static UserReferenceDto toReferenceDto(UserReference entity) {
 			// if user is assigned to a facility, but that facility is not assigned to a district, show no superordinate users. Else, show users of the district (and community) in which the facility is located
 
 			District district = null;
+			Community community = null;
 			List<UserRole> superordinateRoles = UserRole.getWithJurisdictionLevels(superordinateJurisdiction);
 			if (user.getDistrict() != null) {
 				district = districtService.getByReferenceDto(user.getDistrict());
 			} else if (user.getHealthFacility() != null) {
 				district = facilityService.getByReferenceDto(user.getHealthFacility()).getDistrict();
+				community = facilityService.getByReferenceDto(user.getHealthFacility()).getCommunity();
 				superordinateRoles.addAll(UserRole.getWithJurisdictionLevels(JurisdictionLevel.COMMUNITY));
 			}
 
-			superiorUsersList =
-				(district == null)
-					? null
-					: userService.getReferenceList(null, Arrays.asList(district.getUuid()), false, false, true, superordinateRoles);
+			if(district == null) {
+				superiorUsersList = new ArrayList<>();
```

Comment:
could be set to null

Reply:
ticket is asking not to set it to null.

Reply:
you have a null check at the return point of this function, but anyways, the cleaner solution for the null issue would be to initialize  the `superiorUsersList ` with empty list and then you won't need extra branches to change null to empty list.